CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # CHANGELOG
 
+## 1.2.1 - 2017-08-18
```

Comment:
I've published both 1.2.1 and 1.2.0 in my fork, so omitting either one is not a great idea. There's a few places where this has been "shipped" to and nobody knows who else took it.